README.rst

```diff
@@ -41,12 +111,18 @@ Set-up database::
     ./manage.py migrate
     ./manage.py createsuperuser --username admin --email admin@genialis.com
 
-Register processors::
+Register processors
+-------------------
+
+.. code-block::
 
     cd tests
     ./manage.py tool_register --path <path to packages>
```

Comment:
I don't know. I just copied the steps from previous version of the README. @dblenkus, is the Register processes step necessary?